onnxruntime/python/onnxruntime_inference_collection.py

```diff
@@ -486,6 +501,23 @@ def ortvalue_from_shape_and_type(shape=None, element_type=None, device_type='cpu
         return OrtValue(C.OrtValue.ortvalue_from_shape_and_type(shape, element_type,
                         C.OrtDevice(get_ort_device_type(device_type), C.OrtDevice.default_memory(), device_id)))
 
+    @staticmethod
+    def ortvalue_from_data_ptr(shape=None, element_type=None, device_type='cpu', device_id=0, buffer_ptr=None):
```

Comment:
from_dlpack() will also do the job. 
I think Vincent chose this approach, because OrtValue already has a easy way construct a tensor from raw pointer. 
We can certainly complete the dlpack converter and use from_dlpack() for the sake of symmetry. 

I have added an ADO to explore the from_dlpack() appraoch.

Reply:
https://msdata.visualstudio.com/Vienna/_sprints/taskboard/ONNX%20Training/Vienna/Cobalt?fullScreen=true&workitem=1039176